frontend/src/metabase/common/hooks/use-entity-list-query/use-entity-list-query.ts

```diff
@@ -17,8 +17,14 @@ export interface UseEntityListOwnProps<TItem, TQuery = never> {
     state: State,
     options: EntityQueryOptions<TQuery>,
   ) => TItem[] | undefined;
-  getLoading: (state: State, options: EntityQueryOptions<TQuery>) => boolean;
-  getLoaded: (state: State, options: EntityQueryOptions<TQuery>) => boolean;
+  getLoading: (
+    state: State,
+    options: EntityQueryOptions<TQuery>,
+  ) => boolean | undefined;
```

Comment:
The selectors return `undefined` initially and it was one of the reasons for the bug.